workers/unity/Assets/Playground/Scripts/MonoBehaviours/ToggleRotationCommandReceiver.cs

```diff
@@ -0,0 +1,26 @@
+using Generated.Playground;
+using Improbable.Gdk.Core.GameObjectRepresentation;
+using UnityEngine;
+
+namespace Playground.MonoBehaviours
+{
+    public class ToggleRotationCommandReceiver : MonoBehaviour
+    {
+        [Require] private SpinnerRotation.Requirables.CommandRequestHandler requestHandler;
+        private RotationBehaviour rotationBehaviour;
+
+        private void OnEnable()
+        {
+            rotationBehaviour = GetComponent<RotationBehaviour>();
+            if (requestHandler != null) // TODO UTY-791: Needed until prefab preprocessing is implemented, remove as part of UTY-791
+            {
+                requestHandler.OnSpinnerToggleRotationRequest += OnSpinnerToggleRotationRequest;
+            }
+        }
+
+        private void OnSpinnerToggleRotationRequest(SpinnerRotation.SpinnerToggleRotation.ReceivedRequest request)
+        {
+            rotationBehaviour.RotatingClockWise = !rotationBehaviour.RotatingClockWise;
```

Comment:
does it need to respond to the request here, or is that done somewhere else?

Reply:
Responding to a command is not implemented yet. I will extend the API in a subsequent PR so that we can send command responses.